bin/scanpy-compute-graph.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import logging
+from scanpy_wrapper_utils import ScanpyArgParser
+from scanpy_wrapper_utils import read_input_object, write_output_object
+
+
+def main(args):
+    logging.debug(args)
+    import scanpy.api as sc
+
+    adata = read_input_object(args.input_object_file, args.input_format)
+
+    sc.pp.neighbors(adata,
```

Comment:
Where the relation is clear, as here, could you maybe think about naming the scripts for the Scanpy function that's called- 'neighbours' in this case, yes? So scanpy-neighbours.py.